packages/block-editor/src/components/iframe/index.js

```diff
@@ -299,7 +297,6 @@ function Iframe( {
 			}
 
 			return () => {
-				iframeDocument.body.classList.remove( 'is-zoomed-out' );
```

Comment:
We can still remove this right? I saw it's gone from the new changes.

Reply:
no, we still need it for [this](https://github.com/WordPress/gutenberg/blob/058cc37096e567f8e0dfb4dbac92f1038cbda836/packages/block-editor/src/components/block-list/content.scss#L380)

Reply:
Oh, right